xmppserver/src/main/java/org/jivesoftware/openfire/http/HttpSession.java

```diff
@@ -1475,9 +1475,6 @@ public String toString()
             "address=" + getAddress() +
             ", streamID=" + getStreamID() +
             ", status=" + getStatus() +
-            (getStatus() == STATUS_AUTHENTICATED ? " (authenticated)" : "" ) +
-            (getStatus() == STATUS_CONNECTED ? " (connected)" : "" ) +
-            (getStatus() == STATUS_CLOSED ? " (closed)" : "" ) +
```

Comment:
I'm curious about this removal?

Reply:
Previously, `getStatus()` would return a numeric value. To allow humans to make sense of that, that was followed by a text explaining what the numeric value means, by adding a string between parentheses after the numeric value.

With the change in this commit, getStatus will return an enum value, which shows as a human readable text.